.github/workflows/ci.yml

```diff
@@ -142,17 +142,33 @@ jobs:
 
```

Comment:
wondering about permissions and oicd...
TIL: https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-cloud-providers